ds3-sdk/build.gradle

```diff
@@ -57,6 +57,8 @@ task genConfigProperties {
 
         configFile.withWriter{out ->
             out.writeLine("productionBuild=" + productionBuild.toString())
+            out.writeLine("version=" + version.toString())
```

Comment:
The gradle docs list it's type as `Object` so this is the safer option to force it to a string